build.gradle

```diff
@@ -223,8 +225,15 @@ dependencies {
   testCompile group: 'org.springframework.boot', name: 'spring-boot-starter-test'
   testCompile group: 'org.springframework.cloud', name: 'spring-cloud-contract-wiremock'
 
+  testCompile group: 'org.junit.jupiter', name:'junit-jupiter-api', version: versions.junit
+  testCompile group: 'org.junit.jupiter', name:'junit-jupiter-engine', version: versions.junit
+  testCompile group: 'org.junit.vintage', name:'junit-vintage-engine', version: versions.junit
+  testCompile group: 'org.junit.jupiter', name: 'junit-jupiter-params', version: versions.junit
```

Comment:
Is this dependency needed? Do we have parameterised tests?

Reply:
no parameterised tests at the moment but we will hopefully use it... i only changed 1 test at the moment to just start with..